src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -26,6 +26,9 @@ public class OverseasEntitySubmissionDto {
     @JsonProperty("beneficial_owners_government_or_public_authority")
     private List<BeneficialOwnerGovernmentOrPublicAuthorityDto> beneficialOwnersGovernmentOrPublicAuthority;
 
+    @JsonProperty("managing_officers_individual")
+    private List<ManagingOfficerIndividualDto> managingOfficerIndividual;
```

Comment:
Again, variable name should be plural.

Reply:
Yep, but I think should be `managingOfficersIndividual`